WordPress/gutenberg
# Title Bar shifts when switching between saved and save

## What?
Removed `Saved` string from components button text/aria-label


## Why?


Fixes #52295 

## How?


## Testing Instructions





Check the original issue how to replicate the problem but observe now how button width remains the same and layout of header does not shift.

### Testing Instructions for Keyboard


## Screenshots or screencast
